VDA-SBT/contracts/ISoulboundNFT.sol

```diff
@@ -106,9 +106,10 @@ interface ISoulboundNFT {
     /**
      * @notice Get claimed SBT list of user
      * @dev Get the list of msg.sender
+     * @param didAddress address to get the claimed SBT list
      * @return uint[] Claimed token Id list
      */
-    function getClaimedSBTList() external view returns(uint[] memory);
```

Comment:
In the previous version, the `getClaimedSBTList()` function returns the claimed SBT list of the caller.
It uses the `msg.sender` in the smart contract. Here `msg.sender` points the Wallet(MetaMask for example) address that makes this transaction.
Now, the `getClaimedSBTList()` function get the claimer address as a parameter and returns the claimed sbt list of it.